### PR TITLE
ash!mansus grasp now blinds people instead of yeeting them

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

Each heretic path has access to an exclusive (and mandatory if you want to do anything more than create unupgraded eldritch blades) upgrade to the Mansus Grasp spell that the other paths can't get. The one that ash heretics get access to is called "Grasp of Ash".

The Grasp of Ash upgrade causes Mansus Grasp to blind the person you use it on for 10 seconds, blur their vision for 20 seconds, and give them minor nearsightedness (which SHOULD heal on its own, if I'm reading the code right). Essentially, it's a half-duration changeling blind sting effect that's been strapped onto the base effects of the Mansus Grasp spell. This effect replaces the old effect of throwing the person you use your Mansus Grasp on quite a distance away from you.

This does not touch the spell-recharging effect that detonating a blade mark with ash!Mansus Grasp has.

## Why It's Good For The Game

Currently, Grasp of Ash is arguably a DOWNGRADE for the Mansus Grasp spell, as it throws the person you're trying to gank several tiles away from you, giving a not insignificant amount of extra time to respond to being Mansus Grasped (by, say, pulling out a weapon or calling out for help) while you try to get back into melee range. The only reason that the spell is only "arguably" a downgrade is that you can use it in a maintenance tunnel (or, if you're bolder, other areas of the station) to get a wallstun off if you position yourself right, and that reason might disappear if wallstuns get nerfed (and I have a feeling that they will be).

The intent of Grasp of Ash was to turn your Mansus Grasp into a better "get me out of this fight" tool. Thus, the replacement ability (blindness) helps it still fulfill this role (as you can boop someone with ash!Mansus Grasp, then flee the scene with your opponent possibly not even knowing which direction you ran off in), while also not harming Mansus Grasp's primary role of being a fight opener/telebaton+. It also makes ash!Mansus Grasp a bit more useful against hulks (which the other two Mansus Grasp upgrades arguably also do).

## Changelog
:cl: ATHATH
balance: The Grasp of Ash upgrade now causes Mansus Grasp to blind the people you use it on instead of causing it to fling the people you use it on away from you.
/:cl:

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
I'm not sure what to say except 20 seconds of blind is fucking ridiculous. But to compare it to blind sting, there are some costs to using the blind sting, in  being an optional genome costing sting, in costing chemicals, and most importantly being the only effect of the sting. Your mansus grasp does a lot of nasty stuff, while when you come at someone thinking "I am going to blind sting them" you are also thinking "I am not going to knock them down. I am not going to stun them. I am not going to damage them."  You are also thinking "I would rather do this than freeze them after a short period, I would rather do this than transform them, rather do this than cause hallucinations, and by using this I am using some of my limited genomes that could be better spent on those other things if I think they'd be useful in more situations."

The grasp upgrade is part of the package, it isn't the package. In two ways. The grasp itself screws people up even without its special ability- see flesh. And you don't go ash for the grasp bonus, you take it so you can fucking jaunt on a cooldown half the time of regular jaunt. You just get the grasp as a bonus. And since it's supposed to be the easier path, "Grasp that is generally better for getting out of bad situations but has a learning curve which can make it not always the best offensively" is a good design for the throw. Also lol as if getting into melee range is hard on tg with our super fast movespeed and no acceleration. after you grasp someone unless you throw them down a long hallway you can probably  get back into melee range before the cooldown on your next melee swing is up.

**Comment by Rohesie** [comment]
This buff is not looking good.
@EdgeLordExe , do you feel this to be balanced?

**Comment by moocowswag** [comment]
> The intent of Grasp of Ash was to turn your Mansus Grasp into a better "get me out of this fight" tool

I`m probably asking for a lot of work but I`m suggesting this to help you compromise, maybe make this only blind people for full duration if they are facing you?

**Comment by EdgeLordExe** [comment]
@Rohesie i feel like it is a good change

**Comment by EdgeLordExe** [comment]
jist shorten the blind by 10 seconds and it will be fine

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm [hunk 1]
```diff
@@ -41,10 +41,10 @@
...
 			var/obj/effect/proc_holder/spell/targeted/touch/mansus_grasp/MG = X
 			MG.charge_counter = min(round(MG.charge_counter + MG.charge_max * 0.75),MG.charge_max) // refunds 75% of charge.
-	var/atom/throw_target = get_edge_target_turf(C, user.dir)
-	if(!C.anchored)
-		. = TRUE
-		C.throw_at(throw_target, rand(4,8), 14, user)
+	to_chat(C, "<span class='danger'>Your eyes burn horrifically!</span>") //pocket sand! also, this is the message that changeling blind stings use, and no, I'm not ashamed about reusing it
+	C.become_nearsighted(EYE_DAMAGE)
+	C.blind_eyes(10)
+	C.blur_eyes(20)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm:47]: are these times?

Needs time defines if so.

**Comment by Fikou** [comment]
the mansus grasp of ash is really cool now, serves as a retreating thing but its the only one that refunds you time, which makes it cooler than the others in some situations, now its just funny blind you

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm (hunk 1 above)
Comment (ATH1909) [reply code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm:47]: these values are expressed in seconds already

using "SECONDS" would be misleading here, as I'd have to use something like "20 SECONDS" for an effect that lasts for only 2 seconds.

**Comment by ATH1909** [comment]
> jist shorten the blind by 10 seconds and it will be fine

done

**Comment by ATH1909** [comment]
> I'm not sure what to say except 20 seconds of blind is fucking ridiculous. But to compare it to blind sting, there are some costs to using the blind sting, in being an optional genome costing sting, in costing chemicals, and most importantly being the only effect of the sting. Your mansus grasp does a lot of nasty stuff, while when you come at someone thinking "I am going to blind sting them" you are also thinking "I am not going to knock them down. I am not going to stun them. I am not going to damage them." You are also thinking "I would rather do this than freeze them after a short period, I would rather do this than transform them, rather do this than cause hallucinations, and by using this I am using some of my limited genomes that could be better spent on those other things if I think they'd be useful in more situations."
> 
> The grasp upgrade is part of the package, it isn't the package. In two ways. The grasp itself screws people up even without its special ability- see flesh. And you don't go ash for the grasp bonus, you take it so you can fucking jaunt on a cooldown half the time of regular jaunt. You just get the grasp as a bonus. And since it's supposed to be the easier path, "Grasp that is generally better for getting out of bad situations but has a learning curve which can make it not always the best offensively" is a good design for the throw. Also lol as if getting into melee range is hard on tg with our super fast movespeed and no acceleration. after you grasp someone unless you throw them down a long hallway you can probably get back into melee range before the cooldown on your next melee swing is up.

Mate, if you get a mansus grasp off on someone, they're already FUCKED, blindness or no blindness (well, unless they're now on the other side of the room from you). The blindness is here in case you don't have time to finish them off properly (because, say, you're fighting two people at once) and don't want the person you mansus grasp'd to know where you ran off to.

Also, you can just tech over sideways to ash jaunt from another path (which is generally a good idea, as almost every first tier heretic ability is super good for its price). It's not exclusive to people who chose the path of ash as their starting path or anything.

**Pull request merged**